system-tests/.eslintrc.json

```diff
@@ -10,5 +10,8 @@
   },
   "extends": [
     "plugin:@cypress/dev/tests"
-  ]
+  ],
+  "rules": {
+    "no-console": "off"
```

Comment:
I don't think we should be blindly disabling `no-console`. Are we using it that often we can't disable inline? What for if we are?

Reply:
IMO it's ok to do here since these are all tests in this directory - there's no risk of a console log making it to prod. It's used pretty often in `system-tests`, `207 results in 69 files` for `console.`

Reply:
are these specifically in the mocha test files?or also in the mocked projects we are testing?

Reply:
Both. For example, some of the projects do `console.log`s from the plugin so it can be captured in snapshots.

Reply:
This rule catches mistakes fairly often for me in other test files, but I can see the case for turning it off in `system-tests/`. I'm on the fence, but probably fine.